tests/wrappers_test.py

```diff
@@ -71,3 +71,18 @@ def test_plot_rejects_large_input():
 
 def test_plot_ignore_size_disables_size_check():
     pp.plot(np.random.random(1_100_000), ignore_size=True)
+
+
+def test_plot_2d_coord():
+    da = dense_data_array(ndim=2, ragged=True)
+    pp.plot(da)
+    pp.plot(da.transpose())
+
+
+def test_plot_2d_coord_limits():
+    p = pp.plot(dense_data_array(ndim=2, ragged=True))
+    assert np.array_equal(p._ax.get_xlim(), [-0.5, 88.5])
+    assert np.array_equal(p._ax.get_ylim(), [-0.5, 49.5])
+    p = pp.plot(dense_data_array(ndim=2, ragged=True, binedges=True))
+    assert np.array_equal(p._ax.get_xlim(), [0.0, 89.0])
+    assert np.array_equal(p._ax.get_ylim(), [0.0, 50.0])
```

Comment:
Tests like this don't follow what we have, e.g., learned from the "Software Engineering @ Google@ book. This is just checking some numbers, it is unclear what they mean (since we do not see the data setup), and probably the test will break if `dense_data_array` changes somehow?

Reply:
True. So is doing `assert np.array_equal(p._ax.get_xlim(), [da.coords['xx'].min().value, da.coords['xx'].max().value])` better?

Reply:
On second thought, I think I will just remove this test. It sounds like it's going to break all the time, and it in fact has nothing to do with 2d coords, the problem was there also with 1d coords. It's the bin edges which is important.